src/styles.css

```diff
@@ -260,14 +259,14 @@ body {
   'ins ins';
   grid-template-columns: 1fr 2fr;
   grid-template-rows: 1.5fr 2.5fr;
-  height: 80vh;
+  height: 86vh;
 }
 
 .recipe__img {
   grid-area: img;
-  background: url('https://spoonacular.com/recipeImages/595736-556x370.jpg');
+  /* background: url('https://spoonacular.com/recipeImages/595736-556x370.jpg');
```

Comment:
Remove these styles if you aren't using them. Code is cheap 💣

Reply:
Removed :P Might be cheap but emotionally taxing. Catch us on the next episode of code hoarders.